libs/payments/cart/src/lib/checkout.service.ts

```diff
@@ -235,6 +237,12 @@ export class CheckoutService {
     }
     await this.cartManager.finishCart(cart.id, version, {});
 
+    this.statsd.increment('subscription_success', {
+      payment_provider: paymentProvider,
+      offering_id: cart.offeringConfigId,
+      interval: cart.interval,
+    });
+
     // TODO: call sendFinishSetupEmailForStubAccount
     console.log(cart.id, subscription.id);
```

Comment:
nit. not related to this PR, but I suppose we can remove this now?

Reply:
Indeed, looks removable! Thank goodness we're not doing stub accounts anymore.